### ISSUE TITLE
WebdriverManager#start callback not called

### ISSUE THREAD

**Issue by fand** [pose/webdriver-manager#12]
This line seems to be the cause of the problem.
https://github.com/pose/webdriver-manager/blob/master/lib/index.js#L42

I edited the line like this, then it worked.
```javascript
if (/running/g.test(data.toString()) && !started) {
```

But I'm not sure if this works platforms.

Here is my output of `example.js` after fix:
```
$  node example.js
{ standalone: 'not present',
  chrome: 'not present',
  ie: 'not present' }
Updating selenium standalone
downloading http://selenium-release.storage.googleapis.com/2.46/selenium-server-standalone-2.46.0.jar...
Updating chromedriver
downloading http://chromedriver.storage.googleapis.com/2.16/chromedriver_mac32.zip...
chromedriver_2.16.zip downloaded to my-selenium-path/chromedriver_2.16.zip
my-selenium-path/chromedriver_2.16.zip my-selenium-path
selenium-server-standalone-2.46.0.jar downloaded to my-selenium-path/selenium-server-standalone-2.46.0.jar
[ 'my-selenium-path/selenium-server-standalone-2.46.0.jar',
  'my-selenium-path/chromedriver',
  undefined ]
{ standalone: 'ready', chrome: 'ready', ie: 'not present' }
selenium.pid: 290
11:20:45.167 INFO - Launching a standalone Selenium Server
Setting system property webdriver.chrome.driver to my-selenium-path/chromedriver
11:20:45.200 INFO - Java: Oracle Corporation 25.45-b02
11:20:45.201 INFO - OS: Mac OS X 10.10.3 x86_64
11:20:45.210 INFO - v2.46.0, with Core v2.46.0. Built from revision 87c69e2
11:20:45.284 INFO - Driver provider org.openqa.selenium.ie.InternetExplorerDriver registration is skipped:
registration capabilities Capabilities [{ensureCleanSession=true, browserName=internet explorer, version=, platform=WINDOWS}] does not match the current platform MAC
11:20:45.284 INFO - Driver class not found: com.opera.core.systems.OperaDriver
11:20:45.284 INFO - Driver provider com.opera.core.systems.OperaDriver is not registered
11:20:45.371 INFO - RemoteWebDriver instances should connect to: http://127.0.0.1:4444/wd/hub
11:20:45.371 INFO - Selenium Server is up and running
Selenium started. Exiting now.
Attempting to shut down selenium nicely
11:20:45.607 INFO - Command request: shutDownSeleniumServer[, ] on session null
11:20:45.608 INFO - Shutdown command received
11:20:45.609 INFO - initiating shutdown
11:20:45.609 INFO - Got result: OK on session null
11:20:46.111 INFO - Shutting down...
Selenium Standalone has exited with code 0
```

**Comment by pose** [comment]
Thanks @fand for reporting this issue. I've released `6.0.2` that contains a fix for this. Please let me know if it works.

**Comment by fand** [comment]
It works fine! Thx :confetti_ball:
